### PR TITLE
Update manage-proxy.adoc

### PR DESCRIPTION
**Pull request by amagnus**

### CODE REVIEW

File: modules/ROOT/pages/manage-proxy.adoc [hunk 1]
```diff
@@ -7,40 +7,53 @@ This topic describes how to set or update proxy settings used by Runtime Fabric
...
 The output should match the value expected.
 
 == Update the HTTP Proxy
 
-. Obtain the xref:https://runtime-fabric.s3.amazonaws.com/rtfctl/rtfctl-0.2.0[latest version] of `rtfctl` and transfer to a controller node under the `/usr/local/bin` folder with the name `rtfctl`. You may need root access to perform this operation.
+This operation requires two procedures: running a script on each node of your Runtime Fabric to update the cluster, and applying the proxy values to the services running within the Runtime Fabric.
+
+=== Run script on each node
+
+. Copy the `set_dockerd_proxy.sh` from the controller node used to install Runtime Fabric located under `/opt/anypoint/runtimefabric/scripts/` to each controller and worker node under the same location (`/opt/anypoint/runtimefabric/scripts/`).
```
Comment (amagnus) [inline modules/ROOT/pages/manage-proxy.adoc]: should we assume that they have the latest installer here?

Comment (sundippatel) [reply modules/ROOT/pages/manage-proxy.adoc]: Good point. We'll need to provide this script in a location (similar to the Mule license key update script) for customers running on older versions to obtain.

Comment (sundippatel) [reply modules/ROOT/pages/manage-proxy.adoc]: Added a note to guide customers to open a support ticket if unable to locate the script.

**Review by sundippatel — APPROVED** [review]

File: modules/ROOT/pages/manage-proxy.adoc (hunk 1 above)
Comment (amagnus) [reply modules/ROOT/pages/manage-proxy.adoc]: 👍

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**